src/Doctrine/ORM/GiftCardRepository.php

```diff
@@ -69,4 +69,14 @@ public function createAccountListQueryBuilder(CustomerInterface $customer): Quer
 
         return $qb;
     }
+
+    public function findCreatedBefore(\DateTimeInterface $date): array
```

Comment:
```suggestion
    public function findCreatedBefore(\DateTimeInterface $date, int $limit = 100): array
```
Else we could run into memory issues

Reply:
I'd advise to allow null so we can get them all if we want to